test/test-thread.c

```diff
@@ -123,12 +123,14 @@ static void do_work(void* arg) {
     getaddrinfo_do(req);
   }
 
+#ifndef _AIX
```

Comment:
Can you add a comment explaining why this test is disabled on AIX?  LGTM apart from that.